packages/api/src/command/medical/facility/verify-access.ts

```diff
@@ -2,7 +2,24 @@ import { OrganizationBizType } from "@metriport/core/domain/organization";
 import ForbiddenError from "../../../errors/forbidden";
 import { getOrganizationOrFail } from "../organization/get-organization";
 
-export async function verifyCxAccess(cxId: string, throwOnNoAccess = true): Promise<boolean> {
+export async function verifyCxItVendorAccess(
+  cxId: string,
+  throwOnNoAccess = true
+): Promise<boolean> {
```

Comment:
Why do we return anything? It doesnt seem like anywhere where its being called we check the return, except in a test